tools/rostopic/src/rostopic/__init__.py

```diff
@@ -608,7 +608,8 @@ class CallbackEcho(object):
     def __init__(self, topic, msg_eval, plot=False, filter_fn=None,
                  echo_clear=False, echo_all_topics=False,
                  offset_time=False, count=None,
-                 field_filter_fn=None, fixed_numeric_width=None):
+                 field_filter_fn=None, value_transform_fn=None,
+                 fixed_numeric_width=None):
```

Comment:
Please do not change the order of existing arguments but add new arguments at the end.

Same below.